### ISSUE TITLE
[ Request ] docker Image for UI

### ISSUE THREAD

**Issue by MrPresidentWhite** [PLhery/unfollowNinja#85]
Can you please create an docker image for the UI?

**Comment by plhery** [comment]
Yes I may, I'm currently working on a quite big refactor to the API (and UI logic) and can handle that afterwards.
